kv/db_test.go

```diff
@@ -222,8 +222,7 @@ func TestKVDBTransaction(t *testing.T) {
 			t.Errorf("expected value %q; got %q", value, gr.ValueBytes())
 		}
 		return nil
-	})
-	if err != nil {
+	}); err != nil {
```

Comment:
Yes, it can happen for function calls with many arguments, but those rarely reach a dozen lines like this code. You can also narrow the scope by adding `{ ... }` around this section. Or pull the closure out into a named variable.